browser/services/platform.js

```diff
@@ -136,7 +136,7 @@ class Platform {
   static addToUserPath_win32(locations) {
     return Platform.getUserPath_win32().then((pathString)=>{
       let pathLocations = pathString.split(';');
-      return Platform.setUserPath_win32([...locations.filter(item=>!pathLocations.includes(item)), pathString].join(';'));
+      return Platform.setUserPath_win32([...[locations].filter(item=>!pathLocations.includes(item)), pathString].join(';'));
```

Comment:
locations parameter is an array, lets keep it that way. I want to be able to add several locations in single call.